tests/MSBuildDeviceIntegration/Tests/DebuggingTest.cs

```diff
@@ -248,7 +236,6 @@ public override void OnCreate ()
 					EvaluationOptions = EvaluationOptions.DefaultOptions,
 				};
 				options.EvaluationOptions.UseExternalTypeResolver = true;
-				ClearAdbLogcat ();
```

Comment:
Is this needed to clear out any logs from previous runs?

Reply:
`ClearAdbLogcat() ` was added to test setup in https://github.com/xamarin/xamarin-android/commit/b485425dc80c16d22a4a4bb72163a3a062f98e93, so I removed a handful of calls to it in this PR that seemed redundant.